coderd/workspacebuilds.go

```diff
@@ -632,6 +636,11 @@ func (api *API) patchCancelWorkspaceBuild(rw http.ResponseWriter, r *http.Reques
 		})
 		return
 	}
+
+	if !api.publishWorkspaceUpdate(ctx, rw, workspace.ID) {
```

Comment:
Are we sure we want to return an error even if publish failed? I'm mostly asking because I don't know about the failure scenarios of the pubsub, but it seems like we might lie to the client (i.e. cancel was successful, but workspace publish failed).

(Same question for a few other of these.)

Reply:
Hmm, fair enough. It's not fatal enough that we'd want to pass a failure to the client.